app/src/main/java/com/thailam/piggywallet/ui/splash/SplashActivity.java

```diff
@@ -29,10 +32,8 @@ protected void onCreate(Bundle savedInstanceState) {
     public void changeScreen(List<Wallet> wallets) {
         Intent intent = new Intent(this, WalletActivity.class);
         if (wallets != null) {
-            for (Wallet wallet : wallets) {
-                String tag = "wallets" + wallet.getId();
-                intent.putExtra(tag, wallet);
-            }
+            ArrayList<Wallet> list = new ArrayList<>(wallets);
+            intent.putParcelableArrayListExtra(Constants.EXTRA_WALLETS, list);
```

Comment:
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#211-argument-in-activity-and-fragment